includes/integration.php

```diff
@@ -164,18 +170,20 @@ public function ensure_hooks()
         
         public function ensure_identify(){
             // if user is logged in
-            if( !is_admin() && is_user_logged_in() && !( $this->session_get( $this->get_identify_cookie_name() ) ) ){
-                $user       = wp_get_current_user();
-                $user_roles = $user->roles;
-                $this->identify_call_data = $user->user_email;
-                
-                // check if roles should be sent and if they exist
-                if($this->send_roles_as_tags == 'yes' && !empty($user_roles)){
-                    $this->user_tags = $user_roles;
-                    $this->session_set($this->get_identify_cookie_name(), $user_roles);
-                }
-                $this->session_set($this->get_identify_cookie_name(), 'true');
+            if(is_admin() && !is_user_logged_in() && $this->session_get($this->get_identify_cookie_name())){
```

Comment:
Are you sure these should be `&&` and not `||`? I'm now really following what's up here.